fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractSearchDateTest.java

```diff
@@ -112,6 +112,31 @@ public void testSearchDate_date_chained() throws Exception {
 //        assertSearchReturnsComposition("date", "2018-10-29T17:12:00");
         assertSearchDoesntReturnComposition("subject:Basic.date", "2025-10-29");
     }
+    
+    @Test
+    public void testSearchDate_date_or() throws Exception {
+        assertSearchReturnsSavedResource("date", "2018-10-29,9999-01-01");
+        // This returns the target resource multiple times but should not
```

Comment:
maybe make this a separate test and mark as disabled? 
The same for similar ones? like this?  

Just a point of conversation... not to implement.